### PR TITLE
Intro trivy

### PR DESCRIPTION
**Pull request by smythp**
## Type of change

Add introduction to Trivy. 

### How should this PR be tested?

Trivy has a number of interrelated concepts (targets, scanners, etc.), so I used linking between sections more than usual.

I prefer to use code output instead of screenshots, but Trivy's output doesn't display well on our site (tables with very long lines). I added a long alt text to the one table output screenshot.

## Notes

Title feels a little general, but Trivy is used to scan more than images.

Added a blurb on SBOMs that I think will be useful going forward.

Resolves  https://github.com/chainguard-dev/internal/issues/3887

### CODE REVIEW

File: content/chainguard/chainguard-images/working-with-images/scanners/trivy-tutorial/index.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+
+You will receive output similar to the following:
+
+![Screenshot of Trivy output showing first informational messages on which scanners are inabled, then the name of the image, then a listing of issues by severity, and finally a table formatted with ASCII seperators (pipes and underscores) showing itemized issues](trivy_output.png)
+
+### Interpreting Trivy Output
+
+The initial logging portion of Trivy's output indicates which [scanners](#scanners) are enabled and shows warnings if Trivy has an issueperforming the scan. 
+
+In the initial portion of its results output, Trivy summarizes information on the scanned artifact and gives an overview of known vulnerabilities. In the case of a scanned image, the output includes the image digest, a unique hash of the image that can be used as an identifier.
```
Comment (amouat) [inline content/chainguard/chainguard-images/working-with-images/scanners/trivy-tutorial/index.md:191]: I wonder if the last comma should be ";" so it doesn't read like a list.

File: content/chainguard/chainguard-images/working-with-images/scanners/trivy-tutorial/index.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+trivy image -f cyclonedx -o results.cdx.json nginx
+```
+
+Trivy can scan this generated CycloneDX SBOM with the following:
+
+```bash
+trivy sbom results.cdx.json
+```
+
+By default, the `sbom` subcommand scans only for vulnerabilities. License scanning can be enabled using the `--scanners license` flag.
```
Comment (amouat) [inline content/chainguard/chainguard-images/working-with-images/scanners/trivy-tutorial/index.md:167]: Did you try scanning an attestation from a Chainguard image?

**Review by amouat — APPROVED** [review]
Looks good, it's pretty in-depth. 

My comments are all pretty minor.

File: content/chainguard/chainguard-images/working-with-images/scanners/trivy-tutorial/index.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+menu:
+  docs:
+    parent: "scanners"
+weight: 120
+toc: true
+---
+
+[Trivy](https://github.com/aquasecurity/trivy) is a vulnerability scanner for a wide variety of software artifacts and deployments. Trivy is written in the Go programming language and is maintained by [Aqua Security](https://www.aquasec.com/). Trivy targets container images, VMs, filesystems, remote GitHub repositories, and Kubernetes and Amazon Web Services deployments. The tool can be used to detect known vulnerabilities (CVEs), generate SBOMs, analyze licenses, and scan for misconfigurations and exposed secrets.
+
+## Installation
```
[the remark this answers is not in the document]
Comment (smythp) [reply content/chainguard/chainguard-images/working-with-images/scanners/trivy-tutorial/index.md:22]: Listed the options with internal links and moved package managers to the top.

File: content/chainguard/chainguard-images/working-with-images/scanners/trivy-tutorial/index.md [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+
+Scan the image with the following command:
+
+```bash
+trivy image python:3.10.14-alpine3.20
+```
+
+You will receive output similar to the following:
+
+![Screenshot of Trivy output showing first informational messages on which scanners are inabled, then the name of the image, then a listing of issues by severity, and finally a table formatted with ASCII seperators (pipes and underscores) showing itemized issues](trivy_output.png)
```
[the remark this answers is not in the document]
Comment (smythp) [reply content/chainguard/chainguard-images/working-with-images/scanners/trivy-tutorial/index.md:185]: Reran with a container to make sure I had the latest and got the same output. I'll reach out on Slack, for now removed the cutoff bit.

File: content/chainguard/chainguard-images/working-with-images/scanners/trivy-tutorial/index.md (hunk 1 above)
Comment (smythp) [reply content/chainguard/chainguard-images/working-with-images/scanners/trivy-tutorial/index.md:191]: Made a new sentence.

File: content/chainguard/chainguard-images/working-with-images/scanners/trivy-tutorial/index.md (hunk 2 above)
Comment (smythp) [reply content/chainguard/chainguard-images/working-with-images/scanners/trivy-tutorial/index.md:167]: Like this?

```bash
trivy image --sbom-sources rekor --rekor-url https://rekor.sigstore.dev/ cgr.dev/chainguard/nginx
```

Seems to work, but have no other CVE-riddled image to really test on and don't know what other projects actually use rekor.

Comment (smythp) [reply content/chainguard/chainguard-images/working-with-images/scanners/trivy-tutorial/index.md:167]: Got this working, merging.

**Pull request merged**